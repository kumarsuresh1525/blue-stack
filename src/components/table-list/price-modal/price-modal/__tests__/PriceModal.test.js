import React from 'react';
import {render} from '@testing-library/react';
import PriceModal from '../../index';

describe("Price modal test", () => {
  const data = {
    name: 'name',
    image_url: 'image_url',
    createdOn: 12213123123
  }
  const defaultProps = {
    modal: true,
    data,
    toggle: jest.fn(),
    region: 'region',
    className: 'className'
  };
  test("should render price modal", () => {
    const {container, debug} = render(<PriceModal {...defaultProps}/>);
    expect(container).toMatchSnapshot();
  });
});