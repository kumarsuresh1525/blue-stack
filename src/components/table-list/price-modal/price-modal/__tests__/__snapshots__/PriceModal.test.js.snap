// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price modal test should render price modal 1`] = `
<div>
  <div />
</div>
`;
